api_tests/lib/actor.ts

```diff
@@ -106,17 +106,46 @@ export class Actor {
         if (network != "regtest") {
             throw Error("Expected network regtest, found " + network);
         }
-        let type = action.type;
-
-        switch (type) {
+        switch (action.type) {
             case "bitcoin-send-amount-to-address": {
                 action.payload.should.include.all.keys("to", "amount");
                 let { to, amount } = action.payload;
 
                 return this.wallet.btc().sendToAddress(to, parseInt(amount));
             }
             case "bitcoin-broadcast-signed-transaction": {
-                let { hex } = action.payload;
+                action.payload.should.include.all.keys("hex", "locktime");
```

Comment:
isn't it `min_median_time` ?

Reply:
so... is this test useless if it was passing despite being wrong?

Reply:
Fair point, looks pretty useless then :D

Reply:
It actually breaks :D